Sources/Apodini/Configurations/APNSConfiguration.swift

```diff
@@ -0,0 +1,74 @@
+import Vapor
+import APNS
+import JWTKit
+
+
+/// A `Configuration` used for APNS authentication
+public struct APNSConfiguration: Configuration {
+    private let authentication: APNSAuthentication
+    private let topic: String
+    private let environment: APNSwiftConfiguration.Environment
+    
+    
+    /// Creates a new configuration for APNS
+    ///
+    /// - Parameters:
+    ///     - authentication: The APNS authentication
+    ///     - topic: The Bundle identifier of your iOS application
+    ///     - environment: Specifies the APNS environment
+    public init(_ authentication: APNSAuthentication, topic: String, environment: APNSwiftConfiguration.Environment) {
+        self.authentication = authentication
+        self.topic = topic
+        self.environment = environment
+    }
+    
+    public func configure(_ app: Application) {
```

Comment:
Depending on what get's merged first: this or #32 we might habe to do some decoupling here as well